PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -96,14 +96,42 @@
                             "256*filter('hltOverlapFilterIsoMu*PFTau*') + " \
                             "512*filter('hltDoublePFTau*TrackPt1*ChargedIsolation*')"),
             qualityBitsDoc = cms.string("1 = LooseChargedIso, 2 = MediumChargedIso, 4 = TightChargedIso, 8 = TightID OOSC photons, 16 = HPS, 32 = single-tau + tau+MET, 64 = di-tau, 128 = e-tau, 256 = mu-tau, 512 = VBF+di-tau"),            
-        ),   
+        ),
         cms.PSet(
             name = cms.string("Jet"),
             id = cms.int32(1),
-            sel = cms.string("type(85) && pt > 30 && (coll('hltAK4PFJetsCorrected') || coll('hltMatchedVBF*PFJets*PFTau*OverlapRemoval'))"), 
-            l1seed = cms.string("type(-99)"), l1deltaR = cms.double(0.3),
-            l2seed = cms.string("type(85)  && coll('hltAK4CaloJetsCorrectedIDPassed')"),  l2deltaR = cms.double(0.3),
-            qualityBits = cms.string("filter('*CrossCleaned*LooseChargedIsoPFTau*')"), qualityBitsDoc = cms.string("1 = VBF cross-cleaned from loose iso PFTau"),
+            sel = cms.string("( type(0) || type(85) || type(86) || type(-99) )"), 
```

Comment:
what do these additional types correspond to? why was the selection moved from `l1seed` and `l2seed` to here?